### PR TITLE
Add Dart SDK to version info (#4986).

### PR DESCRIPTION
**Pull request by pq**
Adds a new SDK line to the `flutter —version` output.

Sample output:

```
[~/src/git/clones/flutter] (master) $ flutter --version
Flutter on channel master (from git@github.com:pq/flutter.git)
Framework revision ec0f880032 (2 hours ago); engine revision 607d379c23
Dart SDK 1.19.0-dev.5.0
```

### CODE REVIEW

File: packages/flutter_tools/lib/src/cache.dart [hunk 1]
```diff
@@ -76,6 +76,17 @@ class Cache {
     _lock = null;
   }
 
+  static String _dartSdkVersion;
+
+  static String get dartSdkVersion {
+    if (_dartSdkVersion == null) {
+      File versionFile = new File(path.join(flutterRoot, 'bin', 'cache', 'dart-sdk.version'));
```
Comment (abarth) [inline packages/flutter_tools/lib/src/cache.dart]: Maybe https://api.dartlang.org/stable/1.18.1/dart-io/Platform/version.html would be better?

File: packages/flutter_tools/lib/src/version.dart [hunk 2]
```diff
@@ -57,8 +59,9 @@ class FlutterVersion {
   String toString() {
     String from = 'Flutter on channel $channel (from ${repositoryUrl == null ? 'unknown source' : repositoryUrl})';
     String flutterText = 'Framework revision $frameworkRevisionShort ($frameworkAge); engine revision $engineRevisionShort';
+    String dartSdkText = 'Dart SDK $dartSdkVersion';
```
Comment (devoncarew) [inline packages/flutter_tools/lib/src/version.dart]: ==> `Dart version $dartSdkVersion`?

**Comment by devoncarew** [comment]
lgtm; using `Platform.version` makes sense

File: packages/flutter_tools/lib/src/cache.dart (hunk 1 above)
Comment (pq) [inline packages/flutter_tools/lib/src/cache.dart]: Agreed!  Done.

File: packages/flutter_tools/lib/src/version.dart (hunk 2 above)
Comment (pq) [inline packages/flutter_tools/lib/src/version.dart]: Sure.  Done.

**Comment by pq** [comment]
Using `Platform.version`:

```
[~/src/git/clones/flutter] (sdk_version) $ flutter --version
Flutter on channel sdk_version (from git@github.com:pq/flutter.git)
Framework revision e8223e25f0 (3 hours ago); engine revision 607d379c23
Dart version 1.19.0-dev.5.0 (Wed Aug 10 09:29:58 2016) on "macos_x64"
```

**Comment by devoncarew** [comment]
Urk - that version's long. Perhaps trim off anything after the first space?

**Comment by pq** [comment]
Updated to new text and trimmed version.
```
[~/src/git/clones/flutter] (sdk_version) $ flutter --version
Flutter on channel sdk_version (from git@github.com:pq/flutter.git)
Framework revision 41d89d5f6e (3 hours ago); engine revision 607d379c23
Flutter tools using Dart version 1.19.0-dev.5.0
```

**Comment by sethladd** [comment]
lgtm

**Pull request merged**